test/opt/fold_test.cpp

```diff
@@ -4767,4 +4784,4 @@ ::testing::Values(
         3, true)
 ));
 #endif
-}  // anonymous namespace
+}  // anonymous namespace
```

Comment:
Seems odd.

Reply:
I don't know what is going on there.  I tried to undo it, but it did not seem to work.